src/ctap2/commands/get_info.rs

```diff
@@ -140,7 +141,17 @@ pub struct AuthenticatorInfo {
     pub(crate) max_credential_id_length: Option<usize>,
     pub(crate) transports: Option<Vec<String>>,
     pub(crate) algorithms: Option<Vec<PublicKeyCredentialParameters>>,
-    // lots more to come
+    pub(crate) max_ser_large_blob_array: Option<u32>,
```

Comment:
Should the u32 values here be u64? The spec says they're of type "Unsigned Integer" so I think they should be sized for CBOR type 0.

Reply:
I think you are right!